less/home-page.less

```diff
@@ -72,4 +124,86 @@
       }
     }
   }
-}// home
\ No newline at end of file
+
+}// home
+
+
+//Contact content
+
+.intro {
+  @media @phone {
+    img {
+      height: 13rem;
+      width: 100%;
+  }
+}
+}
+
+.inquiry {
+  h2 {
+    margin-bottom: 15px;
+  }
+
+  @media @phone {
+    padding-left: 5%;
+  }
+
+}
+
+form {
+  .displayFlex(space-around, flex-start, column);
+  margin-top: 15px;
+
+  div {
+    margin: 2px;
+  }
+
+  .msg {
+    margin: 5px 0;
+    font-size: 1.2rem;
+  }
+}
+
+.contactinfo {
+  font-size: 1.6rem;
+  padding-bottom: 20px;
+  
+  @media @phone {
+    padding: 0 5%;
```

Comment:
I've found that using percentages on paddings and margins could yield unexpected behaviors in the format of your elements.  Ask Desiree about that.